sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/package-info.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+/**
+ * Package containing classes for TextAnalyticsClient.
+ * The Text Analytics API is a suite of text analytics web services built with
+ * best-in-class Microsoft machine learning algorithms. The API can be used to
+ * analyze unstructured text for tasks such as sentiment analysis, key phrase
+ * extraction and language detection. No training data is needed to use this
+ * API; just bring your text data. This API uses advanced natural language
+ * processing techniques to deliver best in class predictions. Further
+ * documentation can be found in
+ * https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview.
```

Comment:
Remove "en-us". Allow the browser to figure out the language.

Reply:
ahh. This is the auto-generated files as well. Missing this one